names.php

```diff
@@ -333,6 +333,10 @@
   "Ending2-ToaruKagakuNoRailgun.webm" => array(
     "title" => "Ending 2",
     "source" => "Toaru Kagaku no Railgun (A Certain Scientific Railgun)"),
+  "Opening1-ToaruKagakuNoRailgunS.webm" => array(
+    "title" => "Opening 1",
+    "source" => "Toaru Kagaku no Railgun (A Certain Scientific Railgun) S",
+		"subtitles" => "Opening1-ToaruKagakuNoRailgunS.ass"),
```

Comment:
This is mostly just here as an example of the syntax for adding subtitles to names.php